src/site/content/en/learn/css/selectors/index.md

```diff
@@ -0,0 +1,486 @@
+
+{% Aside %}
+If the browser encounters more than one instance of an `id`
+it will still apply any CSS rules that match its selector.
+However, any element that has an `id` attribute is supposed to have a unique value for it,
+so unless you're writing very specific CSS for a single element,
+avoid applying styles with the `id` selector as it means you can't re-use those styles elsewhere.
+{% endAside %}
+
+### Attribute selector
+
+You can look for elements that have a certain HTML attribute,
+or have a certain value for an HTML attribute,
+using the [attribute selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors).
+Instruct CSS to look for attributes by wrapping the selector with square brackets (`[ ]`).
+
+```css
+[data-type="primary"] {
+	color: red;
+}
+```
+
+This CSS looks for all elements that have an attribute of `data-type` with a value of `primary`, like this:
+
+```html
+<div data-type="primary"></div>
+```
+
+Instead of looking for a specific value of `data-type`,
+you can also look for elements with the attribute present, regardless of its value.
+
+```css
+[data-type] {
+	color: red;
+}
+```
+
+```html
+<div data-type="primary"></div>
+<div data-type="secondary"></div>
+```
+
+Both of these `<div>` elements will have red text.
+
+You can use case-sensitive attribute selectors
+by adding an `s` operator to your attribute selector.
+
+```css
+[data-type="primary" s] {
+	color: red;
+}
+```
+
+This means that if a HTML element had a `data-type` of `Primary`,
+instead of `primary`, it would not get red text.
+You can do the opposite—case insensitivity—by using an `i` operator.
+
+Along with case operators,
+you have access to operators that match portions of strings inside attribute values.
+
+```css
+
+/* A href that contains “example.com” */
+[href*="example.com"] {
+	color: red;
+}
+
+/* A href that starts with https */
+[href^="https"] {
+	color: green;
+}
+
+/* A href that ends with .com */
+[href$=".com"] {
+	color: blue;
+}
+```
+
+<figure class="w-figure">
+  {% Codepen {
+    user: 'web-dot-dev',
+    id: 'BapBbOy'
+  } %}
+  <figcaption class="w-figcaption">In this demo, the `$` operator in our attribute selector gets the filetype from the `href` attribute.
+  This makes it possible to prefix the label—based on that filetype—using a pseudo-element.</figcaption>
+</figure>
+
+### Grouping selectors
+
+A selector doesn't have to match only a single element.
+You can group multiple selectors—separating them with commas—like this:
+
+```css
+strong, em, .my-element, [lang] {
+	color: red;
+}
+```
+
+This example extends the color change to both `<strong>` elements and `<em>` elements.
+It's also extended to a class named `.my-class`,and an element that has a `lang` attribute.
+
+## Pseudo-classes and pseudo-elements
+
+CSS provides useful selector types that focus on specific platform state,
+like when an element is hovered, structures _inside_ an element, or parts of an element.
```

Comment:
I had difficulty parsing this sentence. Suggestion would be to break it up:

"CSS provides useful selector types that enabling the styling of specific platform states, like when an element is hovered. You can also style structures _inside_ of an element, or parts of that element."

^ also, what do you mean by structures and parts? can we make that more clear with examples?